File: WebService/WebService/Models/DTO/TableDTO.cs
using Business;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WebService.Models.DTO
{
    public class TableDTO
    {
        public int Id { get; set; }
        public int MaxPeople { get; set; }
        public int Zone_Id { get; set; }
        public bool Empty { get; set; }

        public TableDTO()
        {

        }

        public TableDTO(Table model)
        {
            this.Empty = model.Empty;
            this.Id = model.Id;
            this.MaxPeople = model.MaxPeople;
            this.Zone_Id = model.Zone_Id;
        }
    }
}

File: Mobile_TPV/app/src/main/java/com/joshuaorellana/mobile_tpv/model/business/TableDTO.java
package com.joshuaorellana.mobile_tpv.model.business;

import java.io.Serializable;

/**
 * Created by Joshua-OC on 04/05/2017.
 */

public class TableDTO implements Serializable {

    private int Id;
    private int MaxPeople;
    private boolean Empty;
    private int ZoneId;

    public TableDTO() {
    }

    public TableDTO(int id, int maxPeople, boolean empty, int zoneId) {
        Id = id;
        MaxPeople = maxPeople;
        Empty = empty;
        ZoneId = zoneId;
    }

    public int getId() {
        return Id;
    }

    public void setId(int id) {
        Id = id;
    }

    public int getMaxPeople() {
        return MaxPeople;
    }

    public void setMaxPeople(int maxPeople) {
        MaxPeople = maxPeople;
    }

    public boolean isEmpty() {
        return Empty;
    }

    public void setEmpty(boolean empty) {
        Empty = empty;
    }

    public int getZoneId() {
        return ZoneId;
    }

    public void setZoneId(int zoneId) {
        ZoneId = zoneId;
    }

    @Override
    public String toString() {
        return "TableDTO{" +
                "Id=" + Id +
                ", MaxPeople=" + MaxPeople +
                ", Empty=" + Empty +
                ", ZoneId=" + ZoneId +
                '}';
    }
}
